packages/mint-components/src/components/sqm-image/sqm-image.tsx

```diff
@@ -0,0 +1,42 @@
+import { withHooks } from "@saasquatch/stencil-hooks";
+import { Component, h, Prop, State } from "@stencil/core";
+import { getProps } from "../../utils/utils";
+import { ImageView } from "./sqm-image-view";
+
+/**
+ * @uiName Image Component
+ */
+@Component({
+  tag: "sqm-image",
+  shadow: true,
+})
+export class Image {
+  /**
+   * @uiWidget ImageUpload
+   * @uiName Image Link
+   */
+  @Prop() imageUrl: string;
+
+  /**
+   * @uiName Position Alignment
+   * @uiType string
+   * @uiEnum ["left", "center", "right"]
+   */
+  @Prop() alignment: "left" | "center" | "right" = "center";
+
+  /**
+   * @uiName Background Color
+   * @uiWidget color
+   */
+  @Prop() backgroundColor: string;
+
+  /**
+   * (optional) Helps with constraining minimum size
+   * @uiName Minimum Height
```

Comment:
Would be good to document what some valid inputs would be for the user. I.e They can use percent, pixel etc

Reply:
done!